jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/JavaCodeWhispererFileCrawler.kt

```diff
@@ -21,6 +21,7 @@ import org.jetbrains.jps.model.java.JavaModuleSourceRootTypes
 object JavaCodeWhispererFileCrawler : CodeWhispererFileCrawler() {
     override val fileExtension: String = "java"
     override val testFilenamePattern: Regex = """(?:Test([^/\\]+)\.java|([^/\\]+)Test\.java)$""".toRegex()
+    override val dialects: Set<String> = setOf(".java")
```

Comment:
this one has a dot `.` while the others don't

Reply:
sorry, my miss.